src/components/common/WalletInfo/styles.module.css

```diff
@@ -8,6 +8,37 @@
 .imageContainer {
   display: flex;
   justify-content: center;
+  position: relative;
+}
+
+.walletIcon {
+  position: absolute;
+  display: flex;
+  -webkit-box-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
```

Comment:
What are these -webpack rules? Possible to stick to standard CSS?

Reply:
Oh looks like a copy-paste error. Will remove them